### PR TITLE
Feature/logs to frontend

### PR DESCRIPTION
**Pull request by Jyuart**
For kafka I used this node: [GitHub](https://github.com/confluentinc/cp-docker-images/tree/5.3.3-post/examples/kafka-single-node)
There is only temporary solution on frontend for showing logs in terminal. I can tune it later when we decide on final format for logs.

### CODE REVIEW

File: backend/buildeR.Kafka/KafkaConsumer.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    {
+        private ConsumerConfig _consumerConfig;
+        private string _topic;
+        public IConsumer<Null, string> consumer;
+        public KafkaConsumer(string topic)
+        {
+            _consumerConfig = new ConsumerConfig
+            {
+                GroupId = "logs-consumers-group",
+                BootstrapServers = "localhost:9092",
```
Comment (VoBilyk) [inline backend/buildeR.Kafka/KafkaConsumer.cs:20]: Could you move this to appsettings

File: backend/buildeR.Kafka/KafkaProducer.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        private ProducerConfig _config;
+        private IProducer<Null, string> _producer;
+        private string _topic;
+        public KafkaProducer(string topic)
+        {
+            _topic = topic;
+
+            _config = new ProducerConfig
+            {
+                BootstrapServers = "localhost:9092"
```
Comment (VoBilyk) [inline backend/buildeR.Kafka/KafkaProducer.cs:22]: This one too

File: frontend/src/app/core/services/projects-logs.service.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  private logsHubConnection: HubConnection;
+
+  constructor() {
+    this.buildConnection();
+    this.startConnection();
+   }
+
+   private buildConnection() {
+     this.logsHubConnection = new HubConnectionBuilder()
+     .withUrl('http://localhost:5070/logsHub')
```
Comment (VoBilyk) [inline frontend/src/app/core/services/projects-logs.service.ts:18]: Move it to the enviroments.json

File: frontend/src/app/core/services/projects-logs.service.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProjectLogsService {
+  private logsHubConnection: HubConnection;
+
+  constructor() {
+    this.buildConnection();
+    this.startConnection();
```
Comment (VoBilyk) [inline frontend/src/app/core/services/projects-logs.service.ts:13]: Let's open connection only when you are on on the tab with terminal

File: backend/buildeR.Kafka/KafkaConsumer.cs (hunk 1 above)
Comment (Jyuart) [reply backend/buildeR.Kafka/KafkaConsumer.cs:20]: Should I create appsettings for Kafka class library?

File: backend/buildeR.Common/DTO/ProjectLog.cs [hunk 5]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace buildeR.Common.DTO
+{
+    public class ProjectLog
+    {
+        public DateTime timestamp { get; set; }
```
Comment (Potapy4) [inline backend/buildeR.Common/DTO/ProjectLog.cs:10]: Capitalize field name like `public DateTime Timestamp { get; set; }`

**Pull request merged**